packages/camera/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.5.1
+
+* Can now be compiled with earlier Android sdks when
```

Comment:
I think it's worth saying that this plugin still doesn't actually work on older SDKs and is just a non-op, just to be clear.